pkg/chunk/gcp/fixtures.go

```diff
@@ -21,6 +21,8 @@ const (
 type fixture struct {
 	srv  *bttest.Server
 	name string
+
+	version int
```

Comment:
I'm generally less of a fan of "version" fields, and more of a fan of feature flags, mainly because feature fields are more self-explanatory.